afs-ws/afs-ws-server-utils/src/main/java/com/powsybl/afs/ws/server/utils/AfsSimpleSecurityContext.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.powsybl.afs.ws.server.utils;
+
+import javax.ws.rs.core.SecurityContext;
+import java.security.Principal;
+import java.util.Objects;
+
+/**
+ * @author Paul Bui-Quang <paul.buiquang at rte-france.com>
+ */
+public class AfsSimpleSecurityContext implements SecurityContext {
+
+    private Principal user;
+
+    public AfsSimpleSecurityContext(String username) {
+        this.user = new Principal() {
```

Comment:
I think you should use `UserPrincipal` instead of `Principal`: the methods are exactly the same but this interface is more specific and seems to fit your needs.